test/electionguard/test_election.cpp

```diff
@@ -34,6 +36,35 @@ TEST_CASE("Can serialize CiphertextElectionContext")
     CHECK(fromBson->getManifestHash()->toHex() == context->getManifestHash()->toHex());
 }
 
+TEST_CASE("Version Code in CiphertextElectionContext Matches Constant")
+{
+    // Arrange
+    auto versionCode = string_to_fixed_width_bytes<32>("v2.0.0");
+
+    // Act
+    auto versionElement = bytes_to_q(versionCode, true);
+
+    // Assert
+    CHECK(versionElement->toHex() ==
+          "76322E302E300000000000000000000000000000000000000000000000000000");
+}
+
+TEST_CASE("Parameter Hash in CiphertextElectionContext Matches Constant")
+{
+    // Arrange
+
+    // Act
+    auto context = CiphertextElectionContext::make(3UL, 2UL, TWO_MOD_P().clone(),
+                                                   TWO_MOD_Q().clone(), TWO_MOD_Q().clone());
+
+    // Assert
+    CHECK(context->getParameterHash()->toHex() == PARAMETER_BASE_HASH().toHex());
+
+    // TODO: update the constant check when hasing is updated to E.G. 2.0 spec.
+    // CHECK(context->getParameterHash()->toHex() ==
+    //       "0x2B3B025E50E09C119CBA7E9448ACD1CABC9447EF39BF06327D81C665CDD86296");
```

Comment:
in order to get this to validate we need to modify the internal hashing method as part of #433